app/models/cloud_network.rb

```diff
@@ -7,4 +8,7 @@ class CloudNetwork < ActiveRecord::Base
   has_many   :cloud_subnets, :dependent => :destroy
   has_many   :security_groups
   has_many   :vms
+
+  # Use for virtual columns, mainly for modeling array and hash types, we get from the API
+  serialize :extra_attributes
```

Comment:
Each of the neutron model contains several attributes that are Hash, so any of them if needed. I've removed all the specific attributes for them, so they can be modeled as virtual columns in this one.